libtextsecure/message_receiver.js

```diff
@@ -1545,24 +1546,42 @@ MessageReceiver.prototype.extend({
       window.log.info('Error getting conversation: ', envelope.source);
     }
   },
+  async handleSessionRequestMessage(envelope, content) {
+    const shouldProcessSessionRequest = await session.Protocols.SessionProtocol.shouldProcessSessionRequest(
+      envelope.source,
+      envelope.timestamp
+    );
+
+    if (shouldProcessSessionRequest) {
+      if (content.preKeyBundleMessage) {
+        await this.savePreKeyBundleMessage(
```

Comment:
We might want to actually process the pre key bundle here instead of saving it in the database.
See `handleEndSession` on how that's done

Reply:
Also adding onto this, `handleEndSession` will need to be modified to remove the prekeybundle processing code.

Reply:
done